docs/src/main/sphinx/connector/phoenix.md

```diff
@@ -273,6 +273,7 @@ statements, the connector supports the following features:
 
```

Comment:
Are there any changes between phoenix5 and base-jdbc when you moved this code?

Reply:
I remember in `JdbcPageSourceProvider#getColumnAdaptations` returned a subList of the `columnAdaptationBuilder` while in Phoenix return all the list of `columnAdaptations`